### PR TITLE
Delay all McServ commands

### PR DESCRIPTION
**Pull request by kernle32dll**
Fixes Operation WinBack 2 memory card access

### CODE REVIEW

File: Source/iop/Iop_McServ.cpp [hunk 1]
```diff
@@ -91,7 +92,6 @@ void CMcServ::CountTicks(uint32 ticks, CSifMan* sifMan)
 	moduleData->pendingCommandDelay -= std::min<uint32>(moduleData->pendingCommandDelay, ticks);
 	if(moduleData->pendingCommandDelay == 0)
 	{
-		assert(moduleData->pendingCommand == CMD_ID_GETINFO);
```
Comment (jpd002) [inline Source/iop/Iop_McServ.cpp:94]: We should check that we have a pending command here (ie.: `assert(moduleData->pendingCommand != CMD_ID_NONE`).

**Comment by jpd002** [comment]
Do you think it would make sense to consolidate all delay code in the same place? Basically, get rid of the code that's at the end of `GetInfo` since your change takes care of delaying everything?

If we do that, I think we should also bring in the comments that say which games are sensitive to delays (and also add a note about Operation WinBack 2 in there).

File: Source/iop/Iop_McServ.cpp (hunk 1 above)
Comment (kernle32dll) [reply Source/iop/Iop_McServ.cpp:137]: This could never occur, as there is a `if(moduleData->pendingCommand == CMD_ID_NONE) return;` check above.

**Comment by jpd002** [comment]
Tested plenty of games, seems to be working good. Thanks!

**Pull request merged**